src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -120,7 +124,7 @@ public class InstanceConfiguration implements Describable<InstanceConfiguration>
     private String machineType;
     private String numExecutorsStr;
     private String startupScript;
-    private boolean preemptible;
+    private ProvisioningType provisioningType;
```

Comment:
Note for reviewers: this class is using Lombok (not a good idea!): https://github.com/jenkinsci/google-compute-engine-plugin/blob/8d7f25651b2e2dd68491eca70519725723cbdfea/src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java#L88-L89